README.rst

```diff
@@ -52,6 +52,7 @@ DS1511+   x64        6.2         Yes
 DS1618+   denverton  6.2         Yes
 DS1817+   avoton     6.2         Yes
 DS1815+   avoton     6.2         Yes
+DS1819+   denverton  6.2         Yes (starting with WireGuard-1.0.20210424)
```

Comment:
Did it not work for older versions or did you not try it?

Reply:
I tried it last time last summer and it did not work back then... package and kernel module compiled fine, but actual networking didn't. unfortunately, I didn't save the old version number before deploying the new package, so can't say for sure which version it was back then. 
I could remove the "starting .." text, as anyway someone now coming to the page will try with the newest version.

Reply:
Yeah, remove that and I'll merge this :+1: